easybuild/easyblocks/g/gcc.py

```diff
@@ -383,14 +381,25 @@ def configure_step(self):
         else:
             objdir = self.create_dir("obj")
 
+        build_and_host_options = self.determine_build_and_host_options()
+        configopts += ' ' + ' '.join(build_and_host_options)
+
+        # instead of relying on uname, we use the actual --build option
+        if build_and_host_options and build_and_host_options[0].startswith('--build='):
+            self.platform_lib = build_and_host_options[0].split('=', 1)[1]
```

Comment:
```suggestion
        build_type_option = [x for x in build_and_host_options if x.startswith('--build=')]
        if len(build_type_option) == 1:
            self.platform_lib = build_type_option[0].split('=', 0)
```
make this a bit more robust? or alternatively, let determine_build_and_host_options() return both build and host type options as separate variables?

Reply:
Good point, it was meant to be implicit as suggested by the name but explicit is better: https://github.com/easybuilders/easybuild-easyblocks/pull/2033/commits/7dc3d2cf275e3099e788651d818141b2bd673543